TeslaLib/TeslaClient.cs

```diff
@@ -113,7 +113,81 @@ public TeslaClient(string email, string teslaClientId, string teslaClientSecret,
 
             TeslaAuthHelper = authHelper ?? new TeslaAuthHelper(FlexChargingUserAgent, region);
         }
-        public IOAuthTokenStore OAuthTokenStore { get; set; }
+        public IOAuthTokenStore OAuthTokenStore { get; set; } 
+        public async Task LoginUsingTokenStoreAsync(LoginToken loginToken, bool forceRefreshOlderThanToday = false)
+        {
+            if (string.IsNullOrWhiteSpace(Email))
+            {
+                throw new ArgumentNullException(nameof(Email));
+            }
+            var token = await TokenStoreForThisInstance.GetTokenAsync(Email);

```

Comment:
We already have a token - look at the loginToken parameter.

Reply:
Yes if we already have a token then we update the parameter token else we update it for same account id

Reply:
Thanks. Changes have been push